packages/ui/src/2_molecules/FormGroup/FormGroup.test.tsx

```diff
@@ -0,0 +1,34 @@
+import { render } from '@testing-library/react';
+
+import React from 'react';
+
+import { Input } from '../../1_atoms';
+import { FormGroup } from './FormGroup';
+
```

Comment:
Look at our other tests, you should have `describe` (https://jestjs.io/docs/api#describename-fn) first with the name of the component and then test cases.